mic/imager/loop.py

```diff
@@ -416,6 +416,7 @@ def _stage_final_image(self):
             if item['fstype'] == "ext4":
                 runner.show('/sbin/tune2fs -O ^huge_file,extents,uninit_bg %s '
                             % imgfile)
+                runner.show("/sbin/e2fsck -f -y %s" % imgfile)
```

Comment:
One line up single quotes are used for strings, so I would follow that.

Reply:
Makes sense, fixed.